src/components/SideBar.tsx

```diff
@@ -84,44 +87,123 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const SideBar = (props: SideBarProps) => {
+  dotenv.config();
   const classes = useStyles();
   const theme = useTheme();
+
+  const [userProfileTemplateId, setUserProfileTemplateId] = useState<any>('');
+  const [listOfApps, setListOfApps] = useState<any>([]);
+  const [loadedData, setLoadedData] = useState<any>(false);
+
+  const appsData: any = [...listOfApps];
+
+  const sendUrl = (url: string) => {
+    return request(url);
+  };
+
+  const getProfileTemplateAndAppIds = () => {
+    let baseUrl = `/api/v1/user/types`;
+    axios
+      .all([
+        sendUrl(baseUrl),
+        sendUrl(`/api/v1/apps/user/types?expand=app%2CappLogo&category=apps`),
+      ])
+      .then(
+        axios.spread((...responses) => {
+          let responseOne = responses[0];
+          let responseTwo = responses[1];
+          let defaultId = responseOne!.data[0].id;
+          let appData = responseTwo!.data;
+          setUserProfileTemplateId(defaultId);
+          setListOfApps(appData);
+          setLoadedData(true);
+        })
+      )
+      .catch((errors) => {
+        console.error(errors);
+      });
+  };
+
+  useEffect(() => {
+    getProfileTemplateAndAppIds();
+  }, []);
+
+  let history = useHistory();
+
+  const redirect = (id1: any, id2: any, label: any, logo: string) => {
+    // console.log(id1, id2, label, logo, 'redirect');
+    const logoUrl = encodeURIComponent(logo);
+    history.push(`/mappings/${id1}/${id2}/${label}/${logoUrl}`);
+  };
+
+  const pushAppNames = () => {
+    let apiData: any = [];
+    Object.keys(appsData).forEach((item: any, index: any) => {
+      apiData.push(
+        <>
+          <Divider />
+          <List>
+            <ListItem button>
+              <img
+                style={{
+                  position: 'sticky',
+                  left: '1px',
+                  top: '0px',
+                }}
+                className="appLogo"
+                height="12px"
+                width="auto"
+                src={appsData[index]._embedded.appLogo.href}
```

Comment:
@SaiSrinivasPendyala  we can use item directly instead of appsData[index].

Reply:
Done. Removed Object.values() and chaged it to item.